server/events/project_command_builder.go

```diff
@@ -395,9 +395,9 @@ func (p *DefaultProjectCommandBuilder) buildAllCommandsByCfg(ctx *command.Contex
 		ctx.Log.Info("automatically determined that there were %d projects modified in this pull request: %s", len(modifiedProjects), modifiedProjects)
```

Comment:
Have you tested this out in your own setup and verified it works as expected?

Reply:
Yes, it worked as expected in my setup